docs/resource/k8s.md

```diff
@@ -211,3 +211,10 @@ A CronJob creates Jobs on a repeating schedule.
 <!-- endsemconv -->
 
 [DocumentStatus]: https://github.com/open-telemetry/opentelemetry-specification/tree/v1.22.0/specification/document-status.md
+
+## Well-Known Labels
+
+There are certain Kubernetes [Well-Known Labels and Annotations](https://kubernetes.io/docs/reference/labels-annotations-taints/) that SHOULD align to OpenTelemetry resource attributes. Labels are typically converted to existing OpenTelemetry attributes automatically using an OpenTelemetry Collector component like the [Kubernetes Attribute Processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/k8sattributesprocessor).
+
+* The Well-Known Label `app.kubernetes.io/name` SHOULD align to the OpenTelemetry resource attribute `service.name`.
+* The Well-Known Label `app.kubernetes.io/version` SHOULD align to the OpenTelemetry resource attribute `service.version`.
```

Comment:
This is not a technical nit, but more about organization of semantic conventions in general.

In this PR we K8s resource conventions requiring a certain value for `service.name`, in #312 we have an attempt to [extend the the `service.instance.id` definition with K8s specific data](https://github.com/open-telemetry/semantic-conventions/pull/312/files#diff-a8466589b56b8aefffe0f6d56a338bd7d9d71d3b36d43ca47e2d794adf5d0e34R135).

It would be great to have a consistent way to specify such mappings. I'm slightly in favor of the approach this PR takes (defining values for general attributes in the domain specific conventions).

Reply:
Elaborating a bit on this point, after chatting with @jpkrohling:

Given that both #312 and this PR are merged: if I'd want to instrument a K8s service, the conventions would be spread out across places:
* setting `service.name` is specified in `resource/k8s.md`
* setting `service.version` is specified in `resource/k8s.md`
* setting `service.instance.id` is specified in an algorithm in `resource/README.md`

I think we should think about consistency before merging any of those PRs. Like:
* Should we consistently define algorithms for `service.*`  attributes, like proposed in #312 for `service.instance.id`?
* Should we have consistently have area-specific overrides  for `service.*` attributes, like what's proposed in this PR for `service.name`?
* Or are we fine with having a mix?

We needn't come to a final conclusion before merging anything experimental, but we'd need to before declaring anything stable.